src/SonarLint.Product.props

```diff
@@ -7,5 +7,14 @@
     <RequiresSigning>true</RequiresSigning>
     <SonarQubeTestProject>false</SonarQubeTestProject>
   </PropertyGroup>
+
+  <ItemGroup>
+    <Using Include="System" />
+    <Using Include="System.Collections.Generic" />
+    <Using Include="System.Diagnostics" />
+    <Using Include="System.Linq" />
+    <Using Include="System.Threading" />
+    <Using Include="System.Threading.Tasks" />
```

Comment:
Is there a reason to maintain this list twice, instead of having it once in `Directory.Build.props`?

Reply:
No not really.

I took the [previous PR](https://github.com/SonarSource/sonarlint-visualstudio/pull/5417) as a template and totally missed the `Directory.Build.props`. I'll change it asap!